specs/language/statements.tex

```diff
@@ -1,2 +1,93 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The \texttt{[unroll]} and \texttt{[unroll(n)]} attribute qualifiers can be used
+ to specify that a loop (\texttt{for}, \texttt{while} and \texttt{do while} 
+loops) can be unrolled. This attribute qualifier can be used to specify full 
+unrolling or partial unrolling by a specified amount. This is a compiler hint 
+and the compiler may ignore this directive.
+
+\texttt{n} is the loop unrolling factor and must be a positive integral compile time 
+constant expression. If n is not specified, the compiler determines the 
+unrolling factor for the loop. This attribute is not compatible with the \texttt{[loop]}
+ attribute.
+
+Note:  The \texttt{[unroll]} attribute  must appear immediately before the loop
+ to be affected.
+
+\subsubsection{Unroll Examples}{Stmt.Attr.Unroll.Examples}
```

Comment:
The examples can be informative, but I'm not sure they're needed to illustrate the specification's details.

I think the C++ spec has a way of capturing examples like this in editor notes that are not included in the final specification. That might be a useful feature to add to capture cases like this.

Reply:
Will delete for now,  look into editor notes later.